tests/test_coverage.py

```diff
@@ -0,0 +1,113 @@
+from nose.tools import (
```

Comment:
I added test coverage of ContentServerCoverageProvider before I realized adding test coverage for everything used by IdentifierResolutionMonitor would be a huge job.